EIPS/eip-6110.md

```diff
@@ -76,11 +76,35 @@ Beginning with the `FORK_BLOCK`, each deposit accumulated in the block **MUST**
 in the order they appear in the logs. To illustrate:
 
 ```python
-def parse_deposit_data(deposit_event_data) -> bytes[]:
-  """
-  Parses deposit data from DepositContract.DepositEvent data
-  """
-  pass
+def parse_deposit_data(deposit_event_data):
+    """
+    Parses raw emitted EVM log data to decode the request data
+    """
+
+
+    # These offsets points are data pointers which first encode the 
+    # size of the data (which is always 32 bytes)
+    # The data itself (of this size) is then appended after the size 32 bytes
+    PUBKEY_OFFSET = deposit_event_data[0:32]
```

Comment:
```suggestion
    pubkey_offset = int.from_bytes(deposit_event_data[0:32], byteorder=‘big')
```
Since this is not constant anymore, it shouldn’t be capital case. Plus, i think we should parse an offset as `int` (same for len)